src/interface.d.ts

```diff
@@ -1,4 +1,5 @@
 // Components interfaces
+export * from './global/shared-types/icon.types';
```

Comment:
Not sure if we need to export it?
If we do, why don't we export `link` types for instance?